content/docs/getting-started/technical-documentation/index.md

```diff
@@ -346,6 +185,18 @@ If you would like to nest an image within a list so that it appears as part of t
   ![image2](image2.jpg)
 ```
 
+### Slow techdocs builds
+
+Larger techdocs repositories can take some time to build when being updated or viewed for the first time. 
```

Comment:
We have used both `techdocs` and `tech docs` across these texts. Let's unify the spelling